docs/blog/2019-08-06-gazette-august/index.md

```diff
@@ -0,0 +1,124 @@
+
+### Jason Blitz Lengstorf
+
+Jason has basically been living his life on livestream since the past month. He's done some incredible streams and has many more exciting ones coming up.
+
+My favourite of the past few ones was [Advanced GraphQL techniques in Gatsby](https://www.twitch.tv/videos/462900728) Mikhail Novikov.
+
+### Theme Jam
+
+To celebrate the launch of Gatsby Themes, we announced the [Gatsby Theme Jam](https://themejam.gatsbyjs.org/) last month.
+
+The response was incredible. Seeing so many wonderful themes from all of you has been spectacular and we're afraid we might like _too_ many of them to pick one.
+
+The team is currently hard at work judging and picking their favourites so results should be out very soon!
+
+## Community Highlights
+
+Gatsby has always been a community effort. We get a lot of contributions every month. Want proof?
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We&#39;ve merged in 500 PRs into <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> over the past month! 😱<br><br>Thank you for your contributions, folks! <a href="https://t.co/dsU85CzuGX">pic.twitter.com/dsU85CzuGX</a></p>&mdash; Sid (@chatsidhartha) <a href="https://twitter.com/chatsidhartha/status/1156334013247737856?ref_src=twsrc%5Etfw">July 30, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Some of the highlights from the last month include:
+
+- [lannonbr](https://github.com/lannonbr) [built a Table of Contents component](https://github.com/gatsbyjs/gatsby/pull/15251) that is live on our documentation and is great when reading a lengthy documentation page
+- [kellyvaughn](https://github.com/kellyvaughn) [added support for metafields to product nodes](https://github.com/gatsbyjs/gatsby/pull/16312) in `gatsby-source-shopify`
+- [datakurre](https://github.com/datakurre) added a great [fix to ensure touchNode](https://github.com/gatsbyjs/gatsby/pull/15919) populates typeOwners
+- [grantglidewell](https://github.com/grantglidewell) added [support](https://github.com/gatsbyjs/gatsby/pull/14630) for Gatsby [Preview](https://www.gatsbyjs.com/preview) for Drupal in `gatsby-source-drupal`
+- [elken](https://github.com/elken) added [support for the Prism command-line](https://github.com/gatsbyjs/gatsby/pull/16170) plugin in `gatsby-remark-prismjs`
+- [cephalization](https://github.com/cephalization) added a [great error check](https://github.com/gatsbyjs/gatsby/pull/16272) in Gatsby
+
+Thank you for all your contributions, folks!
+
+## Gatsby Site and Theme of the month
+
+Time to show off some great stuff people have been building
+
+### Site of the Month
+
+The Gatsby site of the month is [www.fourpost.com](https://www.fourpost.com)
+
+We love it because:
+
+- It's so fast
+- The illustrations are so very pretty
+- Uses Gatsby Preview
+- Pulls in content from DatoCMS & EventBrite
+
+### Theme of the Month
```

Comment:
Maybe we should wait until after the Theme Jam results before we start doing a theme of the month?

Reply:
Very very good question

Reply:
I'm siding with @m-allanson on this that I think we should hold off for the theme jam before pushing this. Afterwards I am full on for promoting themes

Reply:
Removed for now